### PR TITLE
Xcorner 22 product carousels

### PR DESCRIPTION
**Pull request by mgbelegu**

### CODE REVIEW

File: assets/scss/components/_carousel.scss [hunk 1]
```diff
@@ -19,3 +18,70 @@
...
+
+    @include bp(medium) {
+        font-size: rem-calc(42px);
+        line-height: rem-calc(56px);
+        margin-bottom: rem-calc(61px);
+    }
+}
+
+$swiper-arrow-left-purple: '../icons/arrow-left-purple.svg';
+$swiper-arrow-right-purple: '../icons/arrow-right-purple.svg';
```
Comment (marcinkrzeminski) [inline assets/scss/components/_carousel.scss]: @mgbelegu please move those to the top of the file

File: templates/components/products/carousel.html [hunk 2]
```diff
@@ -11,8 +11,9 @@
         {{> components/carousel/content-announcement}}
     {{/if}}
 
-    <div>
-        <div class="js-swiper-btn-prev">Previous</div>
-        <div class="js-swiper-btn-next">Next</div>
+    <div class="c-carousel__navigation">
+        <div class="js-swiper-btn-prev c-carousel__btn-prev"></div>
+        <div class="js-swiper-pagination c-carousel__pagination"></div>
+        <div class="js-swiper-btn-next c-carousel__btn-next"></div>
```
Comment (marcinkrzeminski) [inline templates/components/products/carousel.html]: @mgbelegu I usually put those `.js-` prefixed as last selectors inside the `class` attribute. That's a preference, you can ignore if you disagree ;).

File: .eslintcache [hunk 3]
```diff
@@ -1 +1 @@
...ningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1gd3vyx",{"filePath":"11","messages":"12","suppressedMessages":"13","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","suppressedMessages":"16","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/martibelegu/Documents/Xfive/xcorner/assets/js/theme/common/hooks/faceted-search.js",[],[],"/Users/martibelegu/Documents/Xfive/xcorner/assets/js/theme/common/faceted-search.js",[],["17","18"],"/Users/martibelegu/Documents/Xfive/xcorner/assets/js/theme/custom/category-filters.js",[],[],{"ruleId":"19","severity":2,"message":"20","line":236,"column":17,"nodeType":"21","messageId":"22","endLine":236,"endColumn":25,"suppressions":"23"},{"ruleId":"19","severity":2,"message":"20","line":238,"column":17,"nodeType":"21","messageId":"22","endLine":238,"endColumn":25,"suppressions":"24"},"no-param-reassign","Assignment to property of function parameter '$element'.","Identifier","assignmentToFunctionParamProp",["25"],["26"],{"kind":"27","justification":"28"},{"kind":"27","justification":"28"},"directive",""]
```
Comment (marcinkrzeminski) [inline .eslintcache:1]: @mgbelegu I think we can remove this from the repo. Please add it to .gitignore

**Review by marcinkrzeminski — CHANGES_REQUESTED** [review]
@mgbelegu good job sir 👨🏻. A few small adjustments before moving on, please.

**Comment by mgbelegu** [comment]
@marcinkrzeminski Fixed, thank you. Please check again and let me know if I can merge

**Pull request merged**